src/data/actions/grades.js

```diff
@@ -97,6 +110,41 @@ const fetchGrades = (
   }
 );
 
+const formatDate = (serverDate) => {
+  const options = { year: 'numeric', month: 'long', day: 'numeric' };
+  const timeOptions = { hour: '2-digit', minute: '2-digit' };
+  const dt = new Date(serverDate);
+  return `${dt.toLocaleDateString('en-US', options)} at ${dt.toLocaleTimeString('en-US', timeOptions)}`;
+};
+
+// Map from REST response to internal properties
+const formatGradeOverrideForDisplay = historyArray => historyArray.map(item => ({
+  date: formatDate(item.history_date),
+  grader: item.history_user,
+  reason: item.override_reason,
+  adjustedGrade: item.earned_graded_override,
+}));
+
+// Redux action creater wraps web api service and dispatches actions using Redux
+// thunk type. function that returns another function
+// the inner (return) function is called implicitly by redux -thunk layer
+const fetchGradeOverrideHistory = (subsectionId, userId) => (
+  dispatch =>
+    LmsApiService.fetchGradeOverrideHistory(subsectionId, userId)
+      .then(response => response.data)
+      .then((data) => {
+        dispatch(gotGradeOverrideHistory(
+          formatGradeOverrideForDisplay(data.history), data.override.earned_all_override,
+          data.override.possible_all_override, data.override.earned_graded_override,
+          data.override.possible_graded_override,
+        ));
+      })
+      .catch(() => {
```

Comment:
If the Linter lets you, remove this catch block entirely. If it doesnt at least remove the comments.

Reply:
For now, I used the existing pattern of dispatching errorFetching...

Reply:
But why did you do that? It seems as though this changes an errorFetching thing in our store but doesn't result in anything on the UI. Unless I'm missing something here. ...